src/components/Button/Button.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import 'components/Button/button.css';
+
+interface ButtonProps {
+  primary?: boolean;
+  theme?: string;
+  size?: 'small' | 'medium' | 'large' | 'responsive';
+  label: string;
+  onClick?: () => void;
```

Comment:
아직은 사용하지 않더라도 클릭이벤트는 기본적으로 받을 수 있는 인자이니 onClick 함수인자 타입으로 명시해줘도 좋지 않을까요?

```suggestion
  onClick?: (event?: MouseEvent) => void;
```

Reply:
아 제가 신경 쓰지 못한 부분입니다! 
바로 반영 하겠습니다. 앞으로는 아직 사용하지 않더라도 좀 더 신경 쓰며 타입을 지정하겠습니다. 
감사합니다!